src/model/Fact.js

```diff
@@ -0,0 +1,15 @@
+export class Fact {
+  _sentence =
+    "Cat's eyes shine in the dark because of the tapetum, a reflective layer in the eye, which acts like a mirror.";
+  url = 'http://placekitten.com/400/500';
+  alt = 'the animal the sentence is about';
+
+  get sentence() {
+    return this._sentence; 
+  }
+  set sentence(value) {
+    this._sentence = value
+      ? value
+      : "Cat's eyes shine in the dark because of the tapetum, a reflective layer in the eye, which acts like a mirror.";
```

Comment:
This default sentence string is used 2 times. Keep it outside as a variable DEFAULT_FACT

Reply:
Indeed, I was going to do that